avatar/avatar.py

```diff
@@ -27,9 +22,6 @@ async def avatar(self, ctx, *, user: discord.Member=None):
         if not user:
             user = author
 
-        if user.is_avatar_animated():
-            url = user.avatar_url_as(format="gif")
-        if not user.is_avatar_animated():
-            url = user.avatar_url_as(static_format="png")
+        url = user.avatar.with_static_format("png")

```

Comment:
Will this always result in static avatar image? Can it return animated avatar in case user's avatar is an animated image?

Reply:
> Returns a new asset with the specified static format.
>
> **This only changes the format if the underlying asset is not animated. Otherwise, the asset is not changed.**

https://discordpy.readthedocs.io/en/stable/api.html#discord.Asset.with_static_format

Reply:
got it